src/mapeo-project.js

```diff
@@ -840,6 +849,35 @@ async function deleteAll(dataType) {
   return Promise.all(deletions)
 }
 
+/**
+@param {Object} dataTypes
+@param {MapeoProject['translation']} dataTypes.translation
+@param {MapeoProject['preset']} dataTypes.presets
+@param {MapeoProject['field']} dataTypes.fields
+*/
+async function deleteTranslations(dataTypes) {
+  const deletions = []
+  for (const {
+    docId,
+    docIdRef,
+    schemaNameRef,
+  } of await dataTypes.translation.getMany()) {
+    try {
+      if (schemaNameRef === 'presets' || schemaNameRef === 'fields') {
+        const toDelete = await dataTypes[schemaNameRef].getByDocId(docIdRef)
```

Comment:
Could we avoid `await`ing in a loop here? Sketched out an alternate implementation which avoids it (untested):

```js
async function deleteTranslations(dataTypes) {
  const translations = await dataTypes.translation.getMany()

  await Promise.all(translations.map(async (translation) => {
    const { docId, docIdRef, schemaNameRef } = translation

    if (schemaNameRef === 'presets' || schemaNameRef === 'fields') {
      const toDelete = await dataTypes[schemaNameRef].getByDocId(docIdRef)
      if (toDelete.deleted) {
        await dataTypes.translation.delete(docId)
      }
    } else {
      // TODO: log
      await dataTypes.translation.delete(docId)
    }
  }))
}
```